cpp/src/binaryop/compiled/binary_ops.cu

```diff
@@ -413,7 +414,8 @@ void apply_sorting_struct_binary_op(mutable_column_view& out,
   switch (op) {
     case binary_operator::EQUAL: [[fallthrough]];
     case binary_operator::NULL_EQUALS: [[fallthrough]];
-    case binary_operator::NOT_EQUAL:
+    case binary_operator::NOT_EQUAL: [[fallthrough]];
```

Comment:
I'd switch these so that we group "non-null" then "null" operators, rather than "equal" then "not equal" operators.

Reply:
Done.